### PR TITLE
Misc small fixes

### PR DESCRIPTION
**Pull request by mikehearn**

### CODE REVIEW

File: docs/source/inthebox.rst [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
+* **core** containing the core Corda libraries such as crypto functions, types for Corda's building blocks: states,
+  contracts, transactions, attachments, etc. and some interfaces for nodes and protocols.
+* **docs** contains the Corda docsite in restructured text format as well as the built docs in html. The docs can be
+  accessed via ``/docs/index.html`` from the root of the repo.
+* **finance** defines a range of elementary contracts (and associated schemas) and protocols, such as abstract fungible
+  assets, cash, obligation and commercial paper.
+* **gradle** contains the gradle wrapper which you'll use to execute gradle commands.
+* **gradle-plugins** contains some additional plugins which we use to deploy Corda nodes.
+* **lib** contains some dependencies.
+* **node** contains anything specifically required for creating, running and managing nodes (eg: node driver, servlets,
```
Comment (mnesbit) [inline docs/source/inthebox.rst]: We have several new gradle project folders e.g. node-api and node-schemas. Chris's PR is going to add a few more client  subprojects.

Comment (mikehearn) [inline docs/source/inthebox.rst]: Good point - this content was just moved from elsewhere but I updated it a bit.

File: docs/source/index.rst [hunk 2]
```diff
@@ -5,12 +5,12 @@ Welcome to the Corda documentation!
    current state of the code. `Read the docs for milestone release M9.0 <https://docs.corda.net/releases/release-M9.0/>`_.
 
 `Corda <https://www.corda.net/>`_ is an open-source distributed ledger platform. The latest *milestone* (i.e. stable)
-release is M9.0. The codebase is on `GitHub <https://github.com/corda>`_, and our community can be found on
+release is M9.2. The codebase is on `GitHub <https://github.com/corda>`_, and our community can be found on
```
Comment (patrickkuo) [inline docs/source/index.rst]: M9.2 is not out yet, I assume we won't build this doc until M9.2 is out?

Comment (mikehearn) [inline docs/source/index.rst]: I was figuring M9.2 would be out in a day or two, so yeah.

Comment (patrickkuo) [inline docs/source/index.rst]: Cool, we will get M9.2 out tomorrow.

**Pull request merged**